moshi/moshi/models/lm.py

```diff
@@ -30,6 +30,14 @@
 logger = logging.getLogger(__name__)
 
 
+def scatter_with_mask_(tensor: torch.Tensor, dim: int,
```

Comment:
I'm not 100% sure to understand why this would be needed. In the rust implementation, we also do a scatter without mask but it's fine as we don't modify the write positions for the masked rows in this case so the values will be overwritten on the next calls until the row becomes active (and there the cache will be overwritten with legit data). Wouldn't that apply here?

Reply:
I'm not a 100% sure neither... But it works.